app/views/download/_form--help-us-improve.html.haml

```diff
@@ -50,7 +50,7 @@
   %div{ class: "govuk-form-group access-to-data__submit-group"}
     %button{type: 'submit', class: 'govuk-button govuk-!-margin-bottom-0'} Submit
     %span or
-    = link_to "I'd prefer not to say", @next_page, { class: 'js--ga-watch-click' }
+    = link_to "Skip this step", @next_page, { class: 'js--ga-watch-click' }
```

Comment:
very minor, we can use single quotes here as we're not doing string interpolation.

Reply:
Fixed 🔍